web/src/actions/editActivity/costAllocate.js

```diff
@@ -0,0 +1,13 @@
+import {EDIT_APD} from '../editApd';
+
+export const setCostAllocationMethodology = (index, methodology) => ({
+  type: EDIT_APD,
+  path: `/activities/${index}/costAllocationDesc`,
```

Comment:
#1880 changes both of these paths in an effort to make the API and the frontend data structures look the same. Just noting it here so when #1880 is done, I don't forget to come back and update these. 😬